app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/SubmitFeedbackFragment.java

```diff
@@ -46,7 +48,9 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
 
         View view = super.onCreateView(inflater, container, savedInstanceState);
-        ButterKnife.bind(this, view);
+        if (view != null) {
+            ButterKnife.bind(this, view);
```

Comment:
Same as above even if it's not critical..

Reply:
Got that